MAINTAINERS.md

```diff
@@ -54,8 +54,9 @@ If you are an owner of the organization, you should be able to see an automated
 
 ## Organization members
 
-In addition everything that a repository collaborator can do,
-an organization member also has write access to all the repositories in the tldr-pages organization.
+In addition to everything that repository collaborators can do, organization members
+have collaborator access to all the repositories in the tldr-pages organization,
```

Comment:
Hmm, re-reading the previous section's description and this one, I don't think it's necessary to swap "write access" to "collaborator access":

- The previous description spells out more specifically what "write access" weans
- This description already mentions "In addition to everything that repository collaborators can do"

So I'd suggest restoring the "write access" phrasing, to avoid unnecessary repetition.

Reply:
Okay.  I'll do that.  I changed it because it I couldn't tell if it actually meant collaborator access, but then, I didn't read the whole thing like you did.

Reply:
```suggestion
have write access to all the repositories in the tldr-pages organization,
```

Reply:
Those are good points :)